src/archivist/fetcher/fullDomFetcher.js

```diff
@@ -1,18 +1,13 @@
-import config from 'config';
 import puppeteer from 'puppeteer-extra';
 import stealthPlugin from 'puppeteer-extra-plugin-stealth';
 
 import { FetchDocumentError } from './errors.js';
 
 puppeteer.use(stealthPlugin());
 
-const NAVIGATION_TIMEOUT = config.get('fetcher.navigationTimeout');
-const WAIT_FOR_ELEMENTS_TIMEOUT = config.get('fetcher.waitForElementsTimeout');
-const LANGUAGE = config.get('fetcher.language');
-
 let browser;
 
-export default async function fetch(url, cssSelectors) {
+export default async function fetch(url, cssSelectors, options) {
```

Comment:
This function is not exposed outside the `fetcher` module and only be used by it.
The default configuration is defined at the `fetcher` module level to factorize it between `fullDomFetcher` and `htmlOnlyFetcher`, so I think it's enough and there is no necessity to define default values twice or to validate the existence of these variables until this `fetch` function is exposed outside. 
There are many other function in our codebase where required params are not validated because of the additional cost, as long as these functions are not exposed outside their module, it's ok for me. We can have a discussion about that if you want.

Reply:
I agree to rename `options` to `configuration` when contained values are required.

Reply:
Ok, if they are on a sort of “protected” visibility, agreed to not validate. I still think we should convey the requirement through naming 👍

Reply:
👍